OrbitCore/TracepointCustom.h

```diff
@@ -19,5 +22,10 @@ struct EqualTracepointInfo {
     return left.category().compare(right.category()) == 0 && left.name().compare(right.name()) == 0;
   }
 };
+}  // namespace internal
+
+typedef absl::flat_hash_set<orbit_grpc_protos::TracepointInfo, internal::HashTracepointInfo,
+                            internal::EqualTracepointInfo>
+    TracepointInfoSet;
```

Comment:
nit: `using TacepointInfoSet=absl::flat_hash_set ...` instead of typedef

Reply:
Sure. Thank you!